File: Busqueda_renglones-columnas_03-09-25/matriz.js
matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];
console.log("--- Matriz Original ---");
for (let i = 0; i < matriz.length; i++) {
    let fila = "";
    for (let j = 0; j < matriz[i].length; j++) {
        fila += matriz[i][j] ;
    }
    console.log(fila);
}
console.log("\n--- columnas ---");
for (let i = 0; i < matriz[0].length; i++) {
    let columna = 0;
    for (let j = 0; j < matriz.length; j++) {
        columna = matriz[j][i];
process.stdout.write(columna + " ");
    }
}
console.log("\n--- filas ---");
for (let i = 0; i < matriz.length; i++) {
    let fila = 0;
    for (let j = 0; j < matriz[i].length; j++) {
        fila = matriz[i][j] ;
            process.stdout.write(fila + " ");
    }

}


console.log("\nfin del programa");

File: Busqueda_renglones-columnas_03-09-25/Matriz.cs
using System;
using System.Collections.Generic;

public class Matriz
{
    public static void Main(string[] args)
    {
        int[3][3] matriz = {
            {1, 2, 3},
            {4, 5, 6},
            {7, 8, 9}
        };
        
        int filas = matriz.Length;
        int columnas = matriz[0].Length;
        
        Console.WriteLine("--- Matriz Original ---");
        for (int i = 0; i < filas; i++) {
            for (int j = 0; j < columnas; j++) {
                Console.Write(matriz[i][j] + " ");
            }
            Console.WriteLine();
        }

        Console.WriteLine("\n--- columnas ---");
        for (int i = 0; i < columnas; i++) {
            for (int j = 0; j < filas; j++) {
                Console.Write(matriz[j][i] + " ");
            }
        }
        Console.WriteLine("\n--- filas ---");
        for (int i = 0; i < filas; i++) {
            for (int j = 0; j < columnas; j++) {
                Console.Write(matriz[i][j] + " ");
            }
        }
            Console.WriteLine(); 
        

        Console.WriteLine("\nfin del programa");
    }
}